anitya/app.py

```diff
@@ -112,13 +112,17 @@ def after_openid_login(resp):
     login was successful on the OpenID server.
     '''
     default = flask.url_for('index')
+    blacklist = APP.config['BLACKLISTED_USERS']
     if resp.identity_url:
+        next_url = flask.request.args.get('next', default)
         openid_url = resp.identity_url
+        if openid_url in blacklist or resp.email in blacklist:
```

Comment:
"We are very sorry, but due to repeated vandalism, your account has been blocked from logging in to this service."?